README.md

```diff
@@ -1,81 +1,137 @@
 # CakePHP-Sitemap
 
-A CakePHP Plugin for adding automatic XML and HTML Sitemaps to an CakePHP app
-
-
 [![Latest Version](https://img.shields.io/github/release/loadsys/CakePHP-Sitemap.svg?style=flat-square)](https://github.com/loadsys/CakePHP-Sitemap/releases)
+[![Build Status](https://img.shields.io/travis/loadsys/CakePHP-Sitemap/master.svg?style=flat-square)](https://travis-ci.org/loadsys/CakePHP-Sitemap)
+[![Coverage Status](https://img.shields.io/coveralls/loadsys/CakePHP-Sitemap/master.svg?style=flat-square)](https://coveralls.io/r/loadsys/CakePHP-Sitemap)
 [![Software License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](LICENSE.md)
 [![Total Downloads](https://img.shields.io/packagist/dt/loadsys/cakephp_sitemap.svg?style=flat-square)](https://packagist.org/packages/loadsys/cakephp_sitemap)
-<!--
-[![Build Status](https://travis-ci.org/loadsys/{PLUGIN_NAME}.svg?branch=master&style=flat-square)](https://travis-ci.org/loadsys/CakePHP-SocialLinks)
-[![Coverage Status](https://coveralls.io/repos/loadsys/{PLUGIN_NAME}/badge.svg)](https://coveralls.io/r/loadsys/{PLUGIN_NAME})
--->
 
-## Background
+The Sitemap provides a mechanism for display Sitemap style information for a set of Tables that CakePHP has access to.
```

Comment:
* `display` ➡️  `displaying`
* Make `Sitemap` a link to reference material on what "sitemap style information" means?